### PR TITLE
Don't crash setup on utf-8 character codes in ReadMe.

### PR DESCRIPTION
**Pull request by Kelketek**
When I run the setup.py, the installation crashes due to the ReadMe file having UTF-8 character codes.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import setuptools
+import sys
 
-with open("README.md", "r") as fh:
-    long_description = fh.read()
+if sys.version_info.major >= 3:
```
Comment (Pierre-Sassoulas) [inline setup.py]: You can remove the conditional as we do not support python < 3 anymore.

**Comment by Pierre-Sassoulas** [comment]
Nice catch and thank you for the PR ! This was also fixable by having python 3.7 or by settings `export LC_CTYPE=C.UTF-8`  (https://www.python.org/dev/peps/pep-0538/). I already lost some time on this one before elsewhere :)

**Comment by Kelketek** [comment]
Thank you! Happy to help :)

**Comment by Kelketek** [comment]
Please let me know once you've updated the package in the repo-- I have a dependency that's failing to install.

**Pull request merged**